libs/core/src/components/pds-accordion/stories/pds-accordion.stories.js

```diff
@@ -13,21 +13,17 @@ export default {
 
 const BaseTemplate = (args) => html`
 	<pds-accordion component-id="${args.componentId}" open="${args.isOpen}">
-    <pds-row align-items="center" slot="label">
-      <pds-box align-items="center">
-        <pds-icon name="product"></pds-icon>
-        <span style="display: inline-block; margin-left: 8px;">Products</span>
-      </pds-box>
-    </pds-row>
-    <pds-row>
-      <pds-box direction="column" gap="xs" padding="xs">
-        <pds-link variant="plain" href="#">All products</pds-link>
-        <pds-link variant="plain" href="#">Courses</pds-link>
-        <pds-link variant="plain" href="#">Coaching</pds-link>
-        <pds-link variant="plain" href="#">Community</pds-link>
-        <pds-link variant="plain" href="#">Podcasts</pds-link>
-      </pds-box>
-    </pds-row>
+    <pds-box align-items="center" slot="label">
+      <pds-icon name="product"></pds-icon>
+      <span style="display: inline-block; margin-left: 8px;">Products</span>
+    </pds-box>
+    <pds-box direction="column" gap="xs" padding="xs">
+      <pds-link variant="plain" href="#">All products</pds-link>
+      <pds-link variant="plain" href="#">Courses</pds-link>
+      <pds-link variant="plain" href="#">Coaching</pds-link>
+      <pds-link variant="plain" href="#">Community</pds-link>
+      <pds-link variant="plain" href="#">Podcasts</pds-link>
+    </pds-box>
```

Comment:
Did you intend to commit this with this effort?

Reply:
Yes this was on purpose. Just removing an unneeded `pds-row`.